xgi/linalg/laplacian_matrix.py

```diff
@@ -221,16 +228,39 @@ def normalized_hypergraph_laplacian(H, sparse=True, index=False):
             "Every node must be a member of an edge to avoid divide by zero error!"
         )
 
-    D = degree_matrix(H)
-    A, rowdict = clique_motif_matrix(H, sparse=sparse, index=True)
+    (
+        incidence,
+        rowdict,
+        _,  # Discard edge name-index mapping
+    ) = incidence_matrix(H, sparse=sparse, index=True)
+
+    Dv = degree_matrix(H)
+    De = np.sum(incidence, axis=0)
 
     if sparse:
-        Dinvsqrt = csr_array(diags(np.power(D, -0.5)))
+        Dv_invsqrt = csr_array(diags(np.power(Dv, -0.5)))
```

Comment:
```suggestion
        Dv_invsqrt = diags_array(np.power(Dv, -0.5), format="csr")
```
Just saw [this](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.diags_array.html#scipy.sparse.diags_array) does it in one function and seems to be the new scipy go-to. Check that it works